packages/openapi/src/types.ts

```diff
@@ -47,3 +48,12 @@ export type UnPromisify<T> = T extends Promise<infer U> ? U : T;
  * @internal
  */
 export type ConstructorType<T> = new (...args: any[]) => T;
+
+/**
+ * The type for parameter in Custom Request Configuration.
+ * @internal
+ */
+export type CustomRequestConfig = Omit<
```

Comment:
Looks like the correct type, however, youseem to have not added this type to the odata-request-builder, I would assume you will also have to use it there

Reply:
I think the issue is only regarding openapi-generator and not odata generator. Unless I didn't get the full scope of this? I have imported it to `openapi-request-builder` and used in in this class.

Reply:
Okay I see what you mean now after going through the full scope. I will relocate this type in http-client-types and add it in `odata-request-builder` as well.